src/main/resources/db.migration/V1_0__INIT.sql

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
This migration file is located in db.migration folder, but it should be in the db/migration folder otherwise Flyway cannot find it.